### DISCUSSION TITLE
How do I make a discord bot open links?

### DISCUSSION THREAD

**Discussion by AlemPROO** [Rapptz/discord.py#6861] [Q&A]
I was looking for this because I need it to open links for things to work when interacted but like I don't want it to open a certain link I want it to be able for me to give it a link so it opens it, the link loads and then closes it if it is possible please I really need help with that. And if possible to make it so that you type *link -the link- if possible

**Comment by Luc1412** [comment]
Not sure what you plan to do with this link. You could do a normal GET HTTP request to this url. In most cases it would return the plain content of the page eg. HTML, JSON response.

If you need to fully load the page including js you could use something like selenium.

**Reply by AlemPROO** [reply]
Ok, thank you very much I will try.
